src/localizePropsDecorator.js

```diff
@@ -30,30 +38,46 @@ export default function localizePropsDecorator({ containerNamespace, local }) {
       static _instanceCount = 0;
 
       globalizeLocalState = obj => reduce(obj, (acc, v, k) => {
-        const globalKey = local[k] ? `${containerNamespace}${k}` : k;
+        const globalKey = `${containerNamespace}${k}`;
+
+        const { ___local = {} } = this.props;
+
         return {
           ...acc,
-          [globalKey]: local[k] ? {
-            ...this.props[globalKey],
+          [globalKey]: {
+            ...___local[globalKey],
             [this.instanceNamespace]: v
-          } : v
+          }
         };
       }, {});
 
-      localizeLocalState = obj => reduce(obj, (acc, v, k) => {
-        const localKey = k.replace(containerNamespace, '');
-        return {
-          ...acc,
-          [localKey]: local[localKey] && v ? v[this.instanceNamespace] : v
-        };
-      }, {});
+      localizeLocalState = obj => {
+        const state = reduce(obj, (acc, v, k) => {
+          const localKey = k.replace(containerNamespace, '');
+          return {
+            ...acc,
+            [localKey]: local[localKey] && v ? v[this.instanceNamespace] : v
+          };
+        }, {});
+        return state;
```

Comment:
looks like this whole change is mostly a leftover of a debugging `console.log`? seems unneeded